### DISCUSSION TITLE
Looking for Roku SmartHome help.

### DISCUSSION THREAD

**Discussion by YaBoiSp0nge** [jfarmer08/homebridge-wyze-smart-home#89] [Q&A]
Hello,

I was wondering if there is a current plan to use the Roku smart home products? From what I can read the devices themselves are a carbon copy of the Wyze hardware but appear to be going through a different server. Any information you may have is most appreciated.

**Comment by jfarmer08** [comment]
Yes I would like to explore this at some point. I would have to purchase some hardware first. I cannot give a timeframe at this point.

**Reply by jfarmer08** [reply]
I am starting to look at this under https://github.com/jfarmer08/wyze-api if anyone want to to help. 

This looks like a good reference to start. 

https://github.com/SecKatie/wyzeapy/pull/58
